packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -294,8 +294,17 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 			);
 		}
 		for (const name in headers) {
+			let HTTP_prefix = 'HTTP_';
+			/**
+			 * Some headers are special and don't have the HTTP_ prefix.
+			 */
+			if (
+				['content-type', 'content-length'].includes(name.toLowerCase())
+			) {
+				HTTP_prefix = '';
+			}
 			this.addServerGlobalEntry(
-				`HTTP_${name.toUpperCase().replace(/-/g, '_')}`,
+				`${HTTP_prefix}${name.toUpperCase().replace(/-/g, '_')}`,
```

Comment:
does PHP not expose both of these in a normal environment?

Reply:
Surprisingly, not:

```js
fetch(
    'https://playground.wordpress.net/phpinfo.php', 
    { 
        headers: { ["Content-type"]: "text.plain", "Content-length": "132" },
        method: "POST"
    }
)
```

Inspecting in dev tools only yields the HTTP_-less versions.

Reply:
```php
<?php
// display-server.php
var_export( $_SERVER );
```

```bash
php -S 0.0.0.0
```

```bash
curl https://localhost/display-server.php -H 'Content-type: text/plain' -X POST -H 'Host: test.invalid'

array (
  'DOCUMENT_ROOT' => '/path/to/php',
  'REMOTE_ADDR' => '127.0.0.1',
  'REMOTE_PORT' => '54352',
  'SERVER_SOFTWARE' => 'PHP 8.2.6 Development Server',
  'SERVER_PROTOCOL' => 'HTTP/1.1',
  'SERVER_NAME' => '0.0.0.0',
  'SERVER_PORT' => '80',
  'REQUEST_URI' => '/display-server.php',
  'REQUEST_METHOD' => 'POST',
  'SCRIPT_NAME' => '/display-server.php',
  'SCRIPT_FILENAME' => '/path/to/php/display-server.php',
  'PHP_SELF' => '/display-server.php',
  'HTTP_HOST' => 'test.invalid',
  'HTTP_USER_AGENT' => 'curl/8.1.2',
  'HTTP_ACCEPT' => '*/*',
  'CONTENT_TYPE' => 'text/plain',
  'HTTP_CONTENT_TYPE' => 'text/plain',
  'CONTENT_LENGTH' => '0',
  'HTTP_CONTENT_LENGTH' => '0',
  'REQUEST_TIME_FLOAT' => 1685627025.810579,
  'REQUEST_TIME' => 1685627025,
)
```

it seems to show both on my test. compare to `Host` which only gets `HTTP_HOST`